simulation/traffic_simulator/include/traffic_simulator/entity/ego_entity.hpp

```diff
@@ -74,7 +75,7 @@ class EgoEntity : public VehicleEntity
     -> const std::shared_ptr<SimModelInterface>;
 
 public:
-  explicit EgoEntity() = delete;
```

Comment:
Why do you remove explicit?

Reply:
I changed the line due to some build error, which occurred to me once building the SS2 with Autoware. As I checked now, it builds on humble correctly, so i revoked the change. Thank you